docs/guidelines/marketing_guide_to_firefox_l10n.md

```diff
@@ -0,0 +1,164 @@
+| Kind | **Firefox anticipates needs, offering solutions and alternatives where users’ freedom and rights are threatened or abused.**  <br><br>We want the best for our users and the world, so we lead by example. We develop better products, we start dialogues, we work openly and with others, we educate ourselves and others, and we inform our users. In doing so, we act empathically toward all people.|
+| Radical | **Firefox questions the status quo and common practices of tech giants and struggles with confidence for a better internet.**  <br><br>Looking optimistically into the future of the internet is a radical act. Bringing users’ needs before our own is just as radical. We challenge the status quo because we think it’s the right thing to do.|
+
+The Firefox Brand Personality is defined by the interaction of these attributes. Depending on the context, an attribute may be more forward or less forward.
+
+## Tone
+
+Our tone of voice and language choices are:
+
+| Tone | |
+| --- | --- |
+| Direct, clear, easy to understand | We want users to feel welcome, and we use words and concepts that everyone understands.|
+| Concise | We like short headlines, phrases and clear calls-to-action.  We try to avoid complicated sentence structure or sentences without added value.|
+| Authentic | Our copy — like our products — is made by humans for humans. We talk to our users as equals and use natural language.|
+| Natural | We write in active voice and use passive voice sparingly. Avoid nominal style. We prefer two short sentences to one long one.|
+| Respectful | Be respectful without sounding too formal.|
+| Humorous | Be fun without becoming too flippant or cheesy.|
+| Relevant | We meet our users where they are. They should be able to quickly grasp our content and feel a personal connection. Cultural allusions must make sense and fit the market rather than be literally translated without appropriate cultural context.|
+| Optimistic and motivating | We believe in a positive future for the internet. Although we sometimes talk about things that are not going so well, we prefer to take a positive outlook.|
+| Creative | While information and clarity come first, we like to use creative and interesting language in our copy. We never want to sound bland and boring. We try to avoid marketing stereotypes.|
+
+## Glossary of term / Recurring terms (WORK IN PROGRESS)
+
+In case of uncertainties, the [transvision search](https://transvision.mozfr.org/) can also be consulted. However, as the search often shows older versions of translations in the result list (often with the formal address), this tool serves as a guide.
+
+However, the following terms and spellings apply:
```

Comment:
I still think this doesn't make a lot of sense for English. Not a lot of locales will copy and adapt this, and this list should be managed on the Terminology side of things (BWX).